staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -4894,7 +4931,12 @@ func TestDescribeNode(t *testing.T) {
   memory             1Gi (8%)     2Gi (16%)
   ephemeral-storage  0 (0%)       0 (0%)
   hugepages-1Gi      0 (0%)       0 (0%)
-  hugepages-2Mi      512Mi (25%)  512Mi (25%)`}
+  hugepages-2Mi      512Mi (25%)  512Mi (25%)`, `Events:
+  Type    Reason  Age                      From  Message
+  ----    ------  ----                     ----  -------
+  Normal          7y342d                         Node bar status is now: NodeHasNoDiskPressure
+  Normal          7y342d (x2 over 7y342d)        Node bar status is now: NodeReady
```

Comment:
This test now fails because the date is different.

Reply:
> This test now fails because the date is different.

I reworked the test cases